Makefile

```diff
@@ -59,6 +59,12 @@ setup_env:
 	poetry run python -c "import skorch" || true # Details above
 	poetry run python -c "from brevitas.core.scaling import AccumulatorAwareParameterPreScaling" || true # Details above
 
+	# For some issues with Mac Intel
+	if [[ $$(uname) == "Darwin" ]] && [[ $$(uname -m) == 'x86_64' ]]; then \
```

Comment:
is there a fixme to put here ? or at least some comments ? just to have a better context here

btw is the issue known to brevitas ?